setup.py

```diff
@@ -9,7 +9,7 @@
     'luigi',
     'boto3',
     'slack_sdk',
-    'pandas',
+    'pandas<1.2',
```

Comment:
[Ask] Why fix version?

Reply:
This is problem caused by breaking change of pandas 1.2 (RO on 26 Dec.).
When we use `dump`, it will be failed with following output.

I'll fix this on next PR.

```
TypeError: write() argument must be str, not bytes
```